Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -245,6 +246,20 @@ private class ReaderModeButton: UIButton {
 }
 
 private class DisplayTextField: UITextField {
+    weak var accessibilityActionsSource: AccessibilityActionsSource?
+
+    override var accessibilityCustomActions: [AnyObject]! {
+        get {
+            if !editing {
+                return accessibilityActionsSource?.accessibilityCustomActionsForView(self)
+            }
+            return super.accessibilityCustomActions
```

Comment:
Shouldn't need the `if !editing` check anymore since `DisplayTextField` cannot become the first responder. And if you remove that, I think this getter and setter can also be removed, and you can just set the actions directly?